en/modules/openiddict.md

```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
+
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system. 
+* Set **permissions** for clients.
+* Manage **API resources**
+
+See [the module description page](https://commercial.abp.io/modules/Volo.openiddict.Ui) for an overview of the module features.
+
+## How to Install
+
+OpenIddict is pre-installed in [the startup templates](../Startup-Templates/Index) from version 6.0.0-rc1. So, no need to manually install it.
```

Comment:
Give the migration link to users who were using a previous version.

Reply:
https://github.com/abpio/abp-commercial-docs/blob/c211203d84f67da96907debc4bc117df7285064a/en/modules/openiddict.md?plain=1#L13